package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "main": "./src/main/main.js",
   "scripts": {
-    "dev": "gulp && electron-webpack dev",
+    "dev": "NODE_ENV=development gulp && electron-webpack dev",
```

Comment:
@altninja this should not be needed. Electron-webpack will set it when running the dev command. `electron-webpack dev` cheers

Reply:
done